starter-files/starter.html

```diff
@@ -189,15 +189,15 @@ <h1 id="message-label" class="sr-only">
         Message Input
       </h1>
 
-      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="File menu" type="button">
+      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="add a file to this message" type="button">
```

Comment:
How about use the word `attach`, instead of `add` ?
`add a file to this message` => `attach a file to this message`